profiles/models.py

```diff
@@ -0,0 +1,48 @@
+"""
+Models for user profile
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+
+
+class Profile(models.Model):
+    """
+    A user profile
+    """
+    PRIVATE = 'private'
+    PUBLIC_TO_MM = 'public_to_mm'
+    PUBLIC = 'public'
+    ACCOUNT_PRIVACY_CHOICES = (
+        (PRIVATE, 'Private'),
+        (PUBLIC_TO_MM, 'Public to logged in users'),
+        (PUBLIC, 'Public to everyone'),
+    )
+
+    user = models.ForeignKey(User)
+    account_privacy = models.TextField(
```

Comment:
they are the same if you do not put a max_length and it does not make much sense to not doing it when we know the max size of the field, like in this case and many of the following.

anyway feel free to leave it in this way if you have a strong opinion about it.